bayesian/api_v1.py

```diff
@@ -96,6 +97,15 @@ def liveness():
     return jsonify({}), 200
 
 
+@api_v1.route('/generatefile', methods=['POST'])
```

Comment:
To follow the current API naming schema, it would be better to name endpoint `/generate-file`.